### DISCUSSION TITLE
For inner border roundness, how can I say something like use `rounded-lg` but 1 px less

### DISCUSSION THREAD

**Discussion by sedghi** [tailwindlabs/tailwindcss#9630] [Help] [answered]
Since I have double borders, if I use `rounded-lg` on both of them it doesn't work and has some gap because of the border width

![image](https://user-images.githubusercontent.com/7490180/197268219-061b57c2-d390-4e4a-a477-69a7fda26a3d.png)

In order to fix this based on [this answer](https://stackoverflow.com/a/4839726/4125322), I need to put a different radius for the inner one, basically the outer one - its border width. 

So my question is how can I do something like `rounded-lg -1px`? is there such a thing as `rounded-[lg-1px]`?

**Comment by wongjn** [comment, marked as answer]
You would need to "grab" the value that is represented by the `lg` key using `theme()` like: `rounded-[calc(theme(borderRadius.lg)-1px)]`.

**Reply by sedghi** [reply]
Thanks!
